test/SDKMethodRunner_test.py

```diff
@@ -378,55 +358,48 @@ def test_run_job_and_add_log(self, condor_mock):
         # self.assertEqual(self.test_collection.count_documents({}), 0)
 
     def test_get_job_params(self):
-        count = self.test_collection.count_documents({})
-        self.assertEqual(self.test_collection.count_documents({}), count)
-        job_id = self.create_job_rec()
-        self.assertEqual(self.test_collection.count_documents({}), count + 1)
 
-        runner = self.getRunner()
-        params = runner.get_job_params(job_id)
-
-        expected_params_keys = [
-            "wsid",
-            "method",
-            "params",
-            "service_ver",
-            "app_id",
-            "source_ws_objects",
-            "parent_job_id",
-        ]
-        self.assertCountEqual(params.keys(), expected_params_keys)
-        self.assertEqual(params["wsid"], self.ws_id)
-        self.assertEqual(params["method"], "MEGAHIT.run_megahit")
-        self.assertEqual(params["app_id"], "MEGAHIT/run_megahit")
-        self.assertEqual(params["service_ver"], "2.2.1")
-        self.assertCountEqual(params["source_ws_objects"], ["a/b/c", "e/d"])
-        self.assertEqual(params["parent_job_id"], "9998")
-
-        self.test_collection.delete_one({"_id": ObjectId(job_id)})
-        self.assertEqual(self.test_collection.count_documents({}), count)
+        with self.mongo_util.me_collection(self.cfg["mongo-jobs-collection"]):
```

Comment:
You could use `self.mongo_util.mongo_engine_connection` here

Reply:
👍